_examples/aml/main.go

```diff
@@ -7,6 +7,7 @@ import (
 	"strconv"
 
 	yoti "github.com/getyoti/yoti-go-sdk/v3"
+	aml "github.com/getyoti/yoti-go-sdk/v3/aml"
```

Comment:
We don't need these when the package is the same name (same as line above, think I'm removing these in https://github.com/getyoti/yoti-go-sdk/pull/139, so no need to change line above
```suggestion
	"github.com/getyoti/yoti-go-sdk/v3/aml"
```

Reply:
Rebased with your changes